frontend/components/Map.jsx

```diff
@@ -0,0 +1,117 @@
+import 'leaflet/dist/leaflet.css'
+import React, { useState } from 'react'
+import { GeoJSON, MapContainer, useMap } from 'react-leaflet'
+import States from "./map-assets/gz_2010_us_040_00_5m.json"
+import Counties from "./map-assets/gz_2010_us_050_00_5m.json"
+import styles from './map.module.css'
+
+export default function Map() {
+
+    let center = [35, -90]
+    let startingZoom = 4
+    const [selectedState, setSelectedState] = useState(1)
+
+    return (
+        <div className={styles.mappage} >
```

Comment:
If you like, I've found it helpful to deconstruct the applicable `className`s from `styles` (so that you don't have to write `styles.` repeatedly!)